plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveMetadata.java

```diff
@@ -2763,6 +2781,13 @@ public List<SchemaTableName> listViews(ConnectorSession session, Optional<String
                 .collect(toImmutableList());
     }
 
+    @Override
+    public Map<String, Object> getViewProperties(ConnectorSession session, SchemaTableName viewName)
+    {
+        // Extra Properties are not exposed via SHOW CREATE VIEW type of queries
+        return ImmutableMap.of();
```

Comment:
```
// The only currently existing view property (extra_properties) is hidden,
// no need to retrieve its value for SHOW CREATE VIEW
```

Reply:
How do we know `getViewProperties` is used for SHOW CREATE VIEW only / used only on context where hidden properties do not matter?

Maybe the method name could be `getVisibleViewProperties`?

Reply:
Like `MaterializedView#getMaterializedViewProperties` this is used only for show queries. - `PropertyMetadata#isHidden` is not currently used in show queries. We could rename it to `getVisibleViewProperties` but remaining methods like the one used for MV doesn't fit that pattern so do we need to modify them as well ?

Reply:
Add `@Experimental` and merge as is. 

let's rename in a follow-up.